tsconfig.json

```diff
@@ -24,10 +24,11 @@
     "lib": ["ES2015"],
     "paths": {
       "@ukef/*": ["src/*"],
-      "@ukef/constants/*": ["src/constants/index"],
+      "@ukef/constant/*": ["src/constants/index"],
       "@ukef/config/*": ["src/config/*"],
-      "@ukef/helpers/*": ["src/helpers/*"],
+      "@ukef/helper/*": ["src/helpers/*"],
```

Comment:
Is this just a style choice we want to stick to (using singular in import names)?

Reply:
For consistency, I've reverted this back to plural. It should follow the naming conventions of the actual folder